kornia/geometry/transform/image_registrator.py

```diff
@@ -0,0 +1,263 @@
+from typing import Optional, Tuple, Type, Union
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+
+from kornia.geometry.conversions import convert_affinematrix_to_homography
+from kornia.geometry.transform.homography_warper import HomographyWarper
+from kornia.geometry.transform.imgwarp import angle_to_rotation_matrix
+from kornia.geometry.transform.pyramid import build_pyramid
+from kornia.testing import check_is_tensor
+
+__all__ = [
+    "ImageRegistrator",
+    "Homography",
+    "Similarity"]
+
+
+class Homography(nn.Module):
+    def __init__(self):
+        r"""Homography geometric model to be used together with ImageRegistrator
```

Comment:
__init__ docstring should be just after the definition of the class, not in the __init__ method.